org.eclipse.jdt.ui.tests.refactoring/resources/BinaryReferencesWorkspace/BinaryReference/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
This is a test folder, why it is changed?

Reply:
For what I see in [`ValidateProjectEncoding`](https://github.com/eclipse-platform/eclipse.platform/blob/master/resources/bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/resources/ValidateProjectEncoding.java#L76C31-L76C31), the validation is applied in all projects in the workspace without making any distintion as to whether they are test projects or not.

Do you think these changes should be reverted? I think they don't really bother much and the warning just clutters the _Problems_ view, that's why I applied the _Quick Fix_ to all of them.

Reply:
The test projects modified here serve the purpose to be used in tests. If there are no test fails, why is the change needed? Usually, none of the projects here are imported in any development workspace.

Reply:
> If there are no test fails, why is the change needed? 

The idea is to declutter the Problems view and this is a low hanging fruit: it can be done easily with a Quick Fix and AFAICT it doesn't really break anything.

> Usually, none of the projects here are imported in any development workspace.

I installed my IDE following [the official instructions in Contributing.md](https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md#creating-an-eclipse-development-environment) and I got all those projects checked out, _e.g._ these JDT tests
![grafik](https://github.com/eclipse-jdt/eclipse.jdt.ui/assets/2205684/db579365-49a3-4836-95fe-ac781174ced9)

Reply:
@merks : could you please check jdt ui setup? I assume it misses some filters.